ShellScript/Shell-Unix-Generic.sublime-syntax

```diff
@@ -13,6 +13,10 @@ file_extensions:
   - .bash_profile
   - .bash_variables
   - .bashrc
+  - .eslintignore
+  - .npmignore
```

Comment:
These don't appear to be shell scripts, but rather a glob-like syntax. I don't think we should include them.